src/main/java/org/kiwiproject/registry/eureka/common/EurekaInstance.java

```diff
@@ -31,6 +34,12 @@
 @JsonIgnoreProperties(ignoreUnknown = true)
 public class EurekaInstance {
 
+    private static final String COMMIT_REF_FIELD = "commitRef";
```

Comment:
OK we have some common field names between the Consul and Eureka code...should the common things be extracted into a "constants" class? Do we expect them and/or require these specific fields to _always_ exist in Eureka, Consul, and any future service registry?

Reply:
You tell me, these constants were only added because intellij said I was using the string 3 or more times

Reply:
Let's not worry about it for now...